core/src/main/java/org/jclouds/internal/BaseView.java

```diff
@@ -45,7 +45,7 @@ protected BaseView(@Provider Context backend, @Provider TypeToken<? extends Cont
    @SuppressWarnings("unchecked")
    @Override
    public <C extends Context> C unwrap(TypeToken<C> type) {
-      checkArgument(checkNotNull(type, "type").isAssignableFrom(backendType), "backend type: %s not assignable to %s", backendType, type);
+      checkArgument(checkNotNull(type, "type").isSupertypeOf(backendType), "backend type: %s not assignable to %s", backendType, type);
```

Comment:
[minor] Perhaps change the message to `type: %s is not a supertype of %s...` to match the code? Probably more comprehensible for users, too ;-)